main_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestCreatePhoneBook(t *testing.T) {
+	tests := []struct {
+		name            string
+		names, phonenum []string
+		want            map[string]string
+	}{
+		{
+			name:     "empty",
```

Comment:
You may also add a test with multiple duplicate names, e.g. `[]string{"a", "a", "b", "b", "c", "c"}.